metadata.rb

```diff
@@ -18,6 +18,9 @@
 depends 'rightscale_backup', '~> 1.2.0'
 depends 'dns', '~> 0.1.3'
 depends 'git', '~> 4.0.2'
+depends 'aws', '~> 2.9.3'
```

Comment:
A blank install without that line gets 3.3.3 which leads to errors on compile. It's a hard limit on this cookbook so added here